Source/ALS/Private/AlsCharacter.cpp

```diff
@@ -289,6 +296,7 @@ void AAlsCharacter::Tick(const float DeltaTime)
 	RefreshMantling();
 	RefreshRagdolling(DeltaTime);
 	RefreshRolling(DeltaTime);
+	PhysicalAnimation->Refresh(this);
```

Comment:
Should this function really be called from here? UAlsPhysicalAnimationComponent::TickComponent() is always called after AAlsCharacter::Tick(), so if it is moved there, it will still be able to access the most up-to-date data.

Reply:
Initially, I tried to implement it to complete within the ComponentTick of PhysicalAnimation, but that caused flickering, so I changed to the current implementation. I couldn’t eliminate this flickering unless I called PhysicalAnimation::Refresh after RefreshInput(DeltaTime) and before RefreshLocomotionLate(DeltaTime).